rfq/negotiator.go

```diff
@@ -162,11 +162,22 @@ func (n *Negotiator) HandleOutgoingBuyOrder(buyOrder BuyOrder) error {
 		// skip this step.
 		var assetRateHint fn.Option[rfqmsg.AssetRate]
 
-		if n.cfg.PriceOracle != nil {
+		// Construct an asset specifier from the order.
+		// TODO(ffranr): The order should have an asset specifier.
```

Comment:
This TODO is trying to say that `buyOrder` should have an `asset.Specifier` type field rather than constructing an asset specifier at this point. I will clarify the todo in code.